CMakeLists.txt

```diff
@@ -67,6 +66,10 @@ set(BACNET_PROTOCOL_REVISION 19)
 
 find_package(Threads)
 
+if (NOT "${LINUX_SYS}" STREQUAL "AzureSphere")
+  add_library (${PROJECT_NAME} src/bacnet/basic/object/bacfile.c src/bacnet/basic/object/bacfile.h)
```

Comment:
I am getting the following error when trying to build Xpert DS with the dockerfile:

```
CMake Error at CMakeLists.txt:73 (add_library):
  add_library cannot create target "bacnet-stack" because another target with
  the same name already exists.  The existing target is a shared library
  created in source directory "/device-bacnet-c/bacnet-stack".  See
  documentation for policy CMP0002 for more details.
```

I believe this is because we add the ${PROJECT_NAME} library at both line 70 and 73. 
Can be reproduced by cloning device-bacnet-c-private, changing build script to checkout XRT-431-branch and running the dockerize script.

Reply:
@gofling Have pushed a fix.

Reply:
@SteveOss I am getting this instead now:
`CMake Error at CMakeLists.txt:73 (add_library):
  Cannot find source file:

    src/bacnet/basic/object/bacfile.c src/bacnet/basic/object/bacfile.h

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at CMakeLists.txt:73 (add_library):
  No SOURCES given to target: bacnet-stack


CMake Generate step failed.  Build files cannot be regenerated correctly.
The command '/bin/sh -c /device-bacnet-c/scripts/build.sh' returned a non-zero code: 1
`

Reply:
@gofling Looks like is treating both file names due to "". Have committed second fix attempt.